nutkit/frontend/session.py

```diff
@@ -16,7 +16,7 @@ def close(self, hooks=None):
         if not isinstance(res, protocol.Session):
             raise Exception("Should be session but was: %s" % res)
 
-    def run(self, cypher, params=None, tx_meta=None, hooks=None, **kwargs):
+    def run(self, cypher, *, params=None, tx_meta=None, hooks=None, **kwargs):
```

Comment:
Does this `*` mean `extra positional params i don't care`?

Reply:
Almost. `*args` (or any other name after `*`) means `collect extra positional arguments in that variable`. If you just write `*` it means from here on, only keyword arguments. FYI: With Python 3.8 they also introduced `/` wich means up until here, positional arguments only.